xml/System.Net.Http/HttpClient.xml

```diff
@@ -42,38 +42,30 @@ The <xref:System.Net.Http.HttpClient> class instance acts as a session to send H
 
 ### Instancing
 
-<xref:System.Net.Http.HttpClient> is intended to be instantiated once and reused throughout the life of an application. HttpClient is designed to pool connections and reuse a connection across multiple requests. If you instantiate an HttpClient class for every request, the number of sockets available under heavy loads will be exhausted. This exhaustion will result in <xref:System.Net.Sockets.SocketException> errors. Following is an example that uses HttpClient correctly.  
+<xref:System.Net.Http.HttpClient> is intended to be instantiated once and reused throughout the life of an application. HttpClient pools connections and reuses a connection across multiple requests. If you instantiate an HttpClient class for every request, the number of sockets available under heavy loads will be exhausted. This exhaustion will result in <xref:System.Net.Sockets.SocketException> errors.
```

Comment:
As we are touching this -- maybe not just "HttpClient pools connections and reuses a connection across multiple requests." but "HttpClient pools connections *inside the instance* and reuses a connection across multiple requests." (or "inside the *handler* instance") -- as opposed to static pooling that was in .NET Framework?